packages/odyssey-react/src/components/Table/TableContainer.module.scss

```diff
@@ -23,9 +23,9 @@
 }
 
 .title {
```

Comment:
Noticed a "title" here. Happy to swap to all of these to "heading" if that's a refactor we're aiming for.

Reply:
@edburyenegren-okta please do!